ebmdatalab/generate-ebmdatalab-stats.py

```diff
@@ -211,30 +215,30 @@ def main(args):
     plt.rc('figure', titlesize=24)
     fig, axes = plt.subplots(nrows=2, ncols=2, figsize=(20, 12), constrained_layout=True)
 
+    # Plot study type
+    # Only include study types used in >= 5 trials
+    study_type_counts = trials_df['study_type'].value_counts(ascending=True)
+    study_type_counts = study_type_counts[study_type_counts >= 5]
+    ax = study_type_counts.plot(kind='barh', ax=axes[1, 0])
```

Comment:
@rando2 the panel order is hard-coded in the multi-panel figure.  It doesn't matter which order the panels are added to the figure.

Here `ax=axes[1, 0]` is the part that determines which grid position to use for the panel.  You can modify this if you want to rearrange the panels.  The `axes` list has the 0-indexed row number followed by the 0-indexed column number in the grid.

If you do want to change this, I suggest only modifying the .py file.  You don't need to update the images and json locally for such a minor change.  That will make it easier to merge, and we'll get the re-ordered panels the next time the nightly workflow runs.